src/Microsoft.ML.Transforms/CategoricalTransform.cs

```diff
@@ -128,7 +128,7 @@ public Arguments()
         public static IDataView Create(IHostEnvironment env, IDataView input, string name,
             string source = null, OutputKind outputKind = CategoricalEstimator.Defaults.OutKind)
         {
-            return new CategoricalEstimator(env, name, source, outputKind).Fit(input).Transform(input) as IDataView;
+            return new CategoricalEstimator(env, source, name, outputKind).Fit(input).Transform(input) as IDataView;
```

Comment:
Maybe we should consider renaming and re-ordering the parameters to this method as well. `name` and `source` don't really help say what they are. Also, if `new CategoricalEstimator` has it one way, and `CategoricalTransform.Create` has it the other way, that is going to lead to confusion.

Reply:
Sure

---
In reply to: [226108453](https://github.com/dotnet/machinelearning/pull/1252#discussion_r226108453) [](ancestors = 226108453)